3V0L/spice-rack-api
# Add the functionality to upload an image for a recipe

#### What does this PR do?
- Add the functionality to upload an image for a recipe

#### How should this be manually tested?
- Run the server
- Use Insomnia to post a multipart form request containing the following:
```
  title: 'Test',
  time: 10,
  ingredients: 'noodles',
  instructions: {
    "1": "Boil water",
    "2": "Place noodle in water",
    "3": "Drain water",
  },
  servings: 2,
  recipeImage:  Image file(JPEG/PNG)
}
```
- The recipe should be saved
- Use the imageURL provided to send a get request to the server and the image should be returned. e.g. `http://127.0.0.1:3232/uploads/2019-01-28T12:13:41.931ZWhat_image_16.11.43.jpeg`

#### Screenshots
<img width="1144" alt="screenshot 2019-01-28 at 16 12 25" src="https://user-images.githubusercontent.com/33119403/51838520-8ee36d00-2317-11e9-92b2-5b25281199d4.png">
